app/services/maintenance_window.rb

```diff
@@ -0,0 +1,18 @@
+class MaintenanceWindow
+  attr_reader :start, :finish, :now
+
+  def initialize(start:, finish:, now: nil, display_time_zone: 'America/New_York')
+    @start = parse(start, display_time_zone: display_time_zone)
+    @finish = parse(finish, display_time_zone: display_time_zone)
+    @now = now || Time.zone.now
+  end
+
+  def active?
+    (start...finish).cover?(now) if start && finish
+  end
+
+  # @api private
```

Comment:
For my own understanding: What difference / benefit does this offer vs.:

```suggestion
  private
  
```

Reply:
This is a YARD tag: https://www.rubydoc.info/gems/yard/file/docs/Tags.md#api

In Ruby, private is just a suggestion, since you can use `#send` to get around access stuff, so I figure it's easier to just mark this as "hey don't call me", also allows us to test private methods in specs if we wanted. I guess here it's not needed so I'll remove